### DISCUSSION TITLE
When a module is dynamically `import()`ed, would all modules it statically `import` be considered `async` for `splitChunks.chunks`?

### DISCUSSION THREAD

**Discussion by nyngwang** [webpack/webpack#18047] [Q&A] [answered]
Say I `import(A)` and module `A` contains 2 static/plain imports `import B`, `import C`. Then given `splitChunks.chunks` is set to `async`, would `B`, `C` also be selected for optimization?

**Comment by alexander-akait** [comment, marked as answer]
Yes, because they are in async chunk
